app/models/contact.rb

```diff
@@ -0,0 +1,14 @@
+# == Schema Information
+#
+# Table name: contacts
+#
+#  id         :bigint           not null, primary key
+#  email      :string           not null
+#  message    :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Contact < ApplicationRecord
+  validates :email, presence: true
```

Comment:
you can also validate that this has email format, so that users cannot insert a text that is not an email. You can check this [here](https://blog.mailtrap.io/rails-email-validation/)